modules/web/src/app/sshkey/template.html

```diff
@@ -14,7 +14,8 @@
 limitations under the License.
 -->
 
-<mat-card appearance="outlined"
+<mat-card *ngIf="isUserSshKeyEnabled"
```

Comment:
I think it would be better if we redirect the user to project overview or 404 page. Wdyt?

Reply:
we don't do that right now with other pages that can be disable like `externalcluster` or `backubs`

Reply:
or do you suggest to have a guard like `isEnabledPath` that can check if the path is not enable redirect to a new not enabled page?